src/js/components/reviewData/CertificationModal/RevalidateButtons.jsx

```diff
@@ -7,37 +7,40 @@ import React, { PropTypes } from 'react';
 
 const propTypes = {
     closeModal: PropTypes.func,
-    clickedRevalidateButton: PropTypes.func,
-    revalidationThreshold: PropTypes.string
+    clickedRevalidateButton: PropTypes.func
 };
 
 const defaultProps = {
     closeModal: null,
-    clickedRevalidateButton: null,
-    revalidationThreshold: ''
+    clickedRevalidateButton: null
 };
 
 export default class RevalidateButtons extends React.Component {
     render() {
         return (
-            <div>
-                <div className="row">
-                    <div className="col-md-12 mb-10 text-center" id="certify-check">
-                        This submission was created prior to {this.props.revalidationThreshold}.
-                        Please revalidate this submission before certifying it.
-                    </div>
-                </div>
+            <div className="revalidate-content">
+                <h6>Are you sure you want to revalidate your submission?</h6>
+                <p>Revalidation cannot be undone and will trigger the following:</p>
+                <ul>
+                    <li>You will be redirected to the initial submission phase, where Files A, B, and C will
+                        automatically be revalidated.</li>
+                    <li>Files D1, D2, E and F must be regenerated.</li>
+                    <li>Cross-file validations must be run again.</li>
+                </ul>
+                <p>Revalidation will not alter a previously-certified submission, until and if you certify the
```

Comment:
This is really nitpicky, but "until and if" is linguistically vague. This could potentially be interpreted to mean *even* if you certify a revalidated submission, it will not alter a previously certified submission. Can we just change to "unless"? Or maybe I'm misinterpreting...

Reply:
I just copy-pasted the exact requested copy from the ticket